packages/block-library/src/rss/index.php

```diff
@@ -0,0 +1,171 @@
+<?php
+/**
+ * Server-side rendering of the `core/rss` block.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Renders the rss error message.
+ *
+ * @param string $message RSS error message.
+ * @return string|void Void if renders on frontend.
+ */
+function render_rss_error_message( $message ) {
+	return '<div class="components-placeholder"><div class="notice notice-alt notice-error">' . $message . '</div></div>';
+}
+
+/**
+ * Renders the `core/rss` block on server.
+ *
+ * @param array $attributes The block attributes.
+ *
+ * @return string Returns the block content with received rss items.
+ */
+function render_block_core_rss( $attributes ) {
+
+	$url = $attributes['feedURL'];
+
+	if ( empty( $url ) ) {
```

Comment:
My PHP is a bit rusty, but I bet this check should be performed on `$attributes['feedURL']` to prevent notices. It doesn't look like there is a default value provided for this property. Unless it's handled somehow in the code.